src/components/select/select-list/select-list.component.js

```diff
@@ -54,6 +57,7 @@ const SelectList = React.forwardRef(
       flipEnabled = true,
       isOpen,
       multiselectValues,
+      virtualScrollBuffer = React.Children.count(children),
```

Comment:
this was just the simplest way to guarantee that all options are rendered, which I wanted to be the default. (Although arguably it doesn't need to be - I did this when I was concerned virtual scrolling might harm accessibility but Martin seems to think there shouldn't be an issue.)

Bear in mind that the select list is rendered in the DOM (but hidden) even when the select isn't open, so the overscan needs to be the number of children in order to render them all into the DOM in that situation. It may not matter but it seems to work and I'm not sure what your alternative suggestion is (other than just having a fixed default of 10 or something).